tests/cypress/integration/api/workspace.spec.ts

```diff
@@ -0,0 +1,21 @@
+// not an actual test. just demonstrates a test using .graphql/.gql files
```

Comment:
Do we need to worry about testing .gql files?

Reply:
In my opinion, no

Reply:
May we then remove? (Might be confusing for developers)

Reply:
this isn't testing .gql file, it's showing how a test is using them to test/use a graphql api (not the file itself).